README.rst

```diff
@@ -91,6 +91,8 @@ Use the ``--help`` option for more details::
       -u, --username
                             Specify username to use during the upgrade (if not
                             provided, a special user will run the upgrade).
+      -p, --disable-pdb
```

Comment:
There's a convention in the Zope/Plone world of [using `-D, --pdb` to enable post mortem debugging](https://pypi.org/project/zope.testrunner/#some-useful-command-line-options-to-get-you-started).  Would you mind changing this option to match and reversing the condition?

Reply:
Yeah i will look if i can implement it tomorrow. I've opted to use disable rather than enable, cause in previous version it was always active and i didn't wanted to change to much in regards to that, but sure 😃

Reply:
Hi, i've changed the handling now to an enable option.